packages/dm-core-plugins/src/attribute-selector/Sidebar.tsx

```diff
@@ -18,19 +25,27 @@ export const Sidebar = (): JSX.Element => {
           onClick={() => setSelectedTab('home')}
           active={selectedTab === 'home'}
         />
-        {Object.values(childTabs).map((tabData: TChildTab) => (
-          <SideBar.Link
-            key={tabData.attribute}
-            icon={subdirectory_arrow_right}
-            label={
-              tabData.entity?.label ||
-              prettifyName(tabData.entity?.name || '') ||
-              tabData.attribute
-            }
-            onClick={() => setSelectedTab(tabData.attribute)}
-            active={selectedTab === tabData.attribute}
-          />
-        ))}
+        {Object.values(childTabs).map((tabData: TChildTab) => {
+          if (
+            config?.visibleAttributes === undefined ||
+            config?.visibleAttributes.length === 0 ||
+            config?.visibleAttributes.includes(tabData.attribute)
+          ) {
```

Comment:
the visibleAttributes list, specified in the config, must include the attribute name for the sidebar entry to render.

The sidebar entry will also render if the visibleAttributes is an empty list of undefined

Reply:
I wouldn't have the logic here. childTabs is generated in AttributeSelectorPlugin, so I would do a filtering there instead